package.json

```diff
@@ -33,6 +33,7 @@
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
     "sync-externals": "npm run-script sync-schema && npm run-script sync-colors",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && yarn dump-schema -- ../../data && rm -rf node_modules",
+    "sync-schema:localhost": "cd ../metaphysics && yarn dump-schema -- ../emission/data",
```

Comment:
You can just run this from Metaphysics (https://github.com/artsy/metaphysics/blob/80829d09aaab8ff7c5c6d09f18f915c472a7b34d/package.json#L22), but I like this version too!

Reply:
Yeah, this was Orta's idea to make it even easier :)